solutions/confirm_alert.py

```diff
@@ -6,9 +6,7 @@
 
 browser = None
 try:
-    browser = helpers.open_browser_page(
-        link="http://suninjuly.github.io/alert_accept.html"
-    )
+    browser = helpers.open_browser_page("alert_accept")
```

Comment:
Да, получается сложновато...
Почему бы не добавить в helpers строку вида
```python
base_url = "https://suninjuly.github.io"
```
И тогда в py-файлах будет достаточно удобочитаемое и привычное
```suggestion
    browser = helpers.open_browser_page(link=f"{helpers.base_url}/alert_accept.html")
```

В таком случае функцию `open_browser_page` можно не трогать.

Reply:
отлично, согласна, заберу